src/Test.UnitTests.Sarif.Driver/SarifHelpers.cs

```diff
@@ -1,48 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.Sarif.Driver
```

Comment:
>Microsoft [](http://example.com/codeflow?start=10&length=9)

Turns out that test code doesn't use this delegate action injection mechanism highly, so I just ripped it out.

Moving forward we should be updating tests to use the new improved analysis model (which is designed to allow solid testing using the core analyze command mechanism, just as the actual client tool does). So this sort of testing model should wither away, rather than being built on.